core/src/test/scala/com/gu/tableversions/core/TableVersionsSpec.scala

```diff
@@ -187,38 +188,249 @@ trait TableVersionsSpec {
         scenario.unsafeRunSync()
 
       initialTableVersion shouldBe SnapshotTableVersion(Version.Unversioned)
-
-      commitResult1 shouldBe SuccessfulCommit
       currentVersion1 shouldBe tableVersion1
-
-      commitResult2 shouldBe SuccessfulCommit
       currentVersion2 shouldBe currentVersion2
     }
 
+    it should "allow table versions of a partitioned table to be updated from version history" in {
```

Comment:
These are the unit tests for the new functionality (roll back and forward) in `TableVersions`.